.github/testing/grpc_decs.yaml

```diff
@@ -0,0 +1,44 @@
+#!api-testing
+# yaml-language-server: $schema=https://linuxsuren.github.io/api-testing/api-testing-schema.json
+# see also https://github.com/LinuxSuRen/api-testing
+name: grpc-sample
```

Comment:
Currently, it has a bug the testsuite name must be unique. If there are mutiple testsuites have the same name, it will only  run one of them.

There are already two files have the same name:

* https://github.com/LinuxSuRen/api-testing/blob/94228f65c538a6fd65e70ff9bfdfc1902d7231f2/.github/testing/grpc.yaml#L4
* https://github.com/LinuxSuRen/api-testing/blob/94228f65c538a6fd65e70ff9bfdfc1902d7231f2/.github/testing/grpc_ref.yaml#L4

As you can see the following output, it should have 9 records instead of 3.

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/12a65bb3-cb67-459e-a537-ec5ea21a3424)

Reply:
I think all testsuites are executed, because the count of each gRPC api is 3 and column Max, Min, Avg are different.
![image](https://github.com/LinuxSuRen/api-testing/assets/51873347/00d6de84-efc1-46ae-b531-925421ff527d)

Reply:
You are right. I missed the count.